integ-test/src/test/java/org/opensearch/sql/sql/QueryIT.java

```diff
@@ -13,72 +13,66 @@
 import org.opensearch.sql.legacy.SQLIntegTestCase;
 
 public class QueryIT extends SQLIntegTestCase {
-    @Override
-    public void init() throws IOException {
-        loadIndex(Index.BEER);
-    }
+  @Override
+  public void init() throws IOException {
+    loadIndex(Index.BEER);
+  }
 
-    @Test
-    public void all_fields_test() throws IOException {
-        String query = "SELECT * FROM "
-                + TEST_INDEX_BEER + " WHERE query('*:taste')";
-        JSONObject result = executeJdbcRequest(query);
-        assertEquals(16, result.getInt("total"));
-    }
+  @Test
+  public void all_fields_test() throws IOException {
+    String query = "SELECT * FROM " + TEST_INDEX_BEER + " WHERE query('*:taste')";
+    JSONObject result = executeJdbcRequest(query);
+    assertEquals(16, result.getInt("total"));
+  }
 
-    @Test
-    public void mandatory_params_test() throws IOException {
-        String query = "SELECT Id FROM "
-                + TEST_INDEX_BEER + " WHERE query('Tags:taste OR Body:taste')";
-        JSONObject result = executeJdbcRequest(query);
-        assertEquals(16, result.getInt("total"));
-    }
+  @Test
+  public void mandatory_params_test() throws IOException {
+    String query = "SELECT Id FROM " + TEST_INDEX_BEER + " WHERE query('Tags:taste OR Body:taste')";
+    JSONObject result = executeJdbcRequest(query);
+    assertEquals(16, result.getInt("total"));
+  }
 
-    @Test
-    public void all_params_test() throws IOException {
-        String query = "SELECT Id FROM " + TEST_INDEX_BEER
-                + " WHERE query('Tags:taste', escape=false,"
-                + "allow_leading_wildcard=true, enable_position_increments=true,"
-                + "fuzziness= 1, fuzzy_rewrite='constant_score', max_determinized_states = 10000,"
-                + "analyzer='standard', analyze_wildcard = false, quote_field_suffix = '.exact',"
-                + "auto_generate_synonyms_phrase_query=true, boost = 0.77,"
-                + "quote_analyzer='standard', phrase_slop=0, rewrite='constant_score', type='best_fields',"
-                + "tie_breaker=0.3, time_zone='Canada/Pacific', default_operator='or',"
-                + "fuzzy_transpositions = false, lenient = true, fuzzy_max_expansions = 25,"
-                + "minimum_should_match = '2<-25% 9<-3', fuzzy_prefix_length = 7);";
-        JSONObject result = executeJdbcRequest(query);
-        assertEquals(8, result.getInt("total"));
-    }
+  @Test
+  public void all_params_test() throws IOException {
+    String query =
+        "SELECT Id FROM "
+            + TEST_INDEX_BEER
+            + " WHERE query('Tags:taste', escape=false,allow_leading_wildcard=true,"
```

Comment:
Can you reformat this too?

Reply:
Don't believe we can, as next spotlessApply will reformat the string.